### PR TITLE
User Dash Project List

### PR DESCRIPTION
**Pull request by agordon1123**
# Description
- Wired up Labs API to user dashboard
- Completed ProjectList.js and Project.js

Fixes # (issue)
- Will fix app breaking bugs

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] This change requires a documentation update

## Change Status

- [X] Complete, but not tested (may need new tests)

# How Has This Been Tested?

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: mission-control-client/src/components/dashboard/projects/Project.js [hunk 1]
```diff
@@ -5,16 +5,16 @@ const Project = props => {
 
   return (
     <div
-      onClick={() => props.setActiveProduct(props.el)}
+      onClick={() => props.setActiveProduct(props.el.project)}
       id={
```
Comment (danbergelt) [inline mission-control-client/src/components/dashboard/projects/Project.js:9]: @agordon1123 should this be props.setActiveProject?

File: mission-control-client/src/components/dashboard/projects/Project.js [hunk 2]
```diff
@@ -5,16 +5,16 @@ const Project = props => {
...
         props.active.id && props.active.id === props.i
           ? "product-active" : undefined
       }
       className="product-container"
     >
       <div className="product-description">
-        <h3 className="product-title">{props.el.name}</h3>
-        {props.i.split('').pop() % 2 === 0 ? (
+        <h3 className="product-title">{props.active.name}</h3>
+        {props.active.end ? (
```
Comment (danbergelt) [inline mission-control-client/src/components/dashboard/projects/Project.js]: @agordon1123 How is this tracking whether a project is ended or not?

File: mission-control-client/src/components/dashboard/projects/Project.js (hunk 1 above)
Comment (Artyk77-zz) [reply mission-control-client/src/components/dashboard/projects/Project.js:9]: I am confused as well with that

**Review by BDesselle — DISMISSED** [review]

File: mission-control-client/src/components/dashboard/projects/Project.js (hunk 1 above)
Comment (agordon1123) [reply mission-control-client/src/components/dashboard/projects/Project.js:9]: Correction made. Thank you!

**Review by Artyk77-zz — DISMISSED** [review]

**Review by dionys93 — DISMISSED** [review]

**Review by BDesselle — DISMISSED** [review]

**Review by dionys93 — APPROVED** [review]

**Review by danbergelt — APPROVED** [review]

**Review by BDesselle — APPROVED** [review]

**Pull request merged**